pages/vi/vi-github-and-markdown.md

```diff
@@ -102,7 +102,7 @@ Then, you can create a pull request by clicking the `Pull requests` button and t
 
 Finally, click `Create pull request` and someone from our team will review the changes.   
 
- ![Complete Pull Request](images/vi-create-pull-request.png)
+ ![Complete Pull Request](images/vi-complete-pullrequest-image.jpg)
```

Comment:
Forgot to mention that there's already an open PR #1331 updating this screenshot. So this change is not necessary.

Reply:
Okay I will not make changes to this screenshot and will revert it back. Thanks